docs/SUMMARY.md

```diff
@@ -21,6 +21,7 @@
 * [Materialization](Advanced Features/Materialization.md)
 * [Working with Manifests](Advanced Features/Working with Manifests.md)
 * [S3 Select](Advanced Features/S3 Select.md)
+* [Workflows](Advanced Features/workflows.md)
```

Comment:
Very much looking for a .md file that shows an example of workflow file in this PR :)